Solutions/Tomcat/Hunting Queries/TomcatUncommonUAsWithClientErrors.yaml

```diff
@@ -0,0 +1,27 @@
+id: 033d672c-4882-11ec-81d3-0242ac130003
+name: Tomcat - Rare user agents with client errors
+description: |
+  'Query shows rare user agent strings with client errors'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  TomcatEvent
+  | where TimeGenerated > ago(24h)
+  | where isnotempty(HttpUserAgentOriginal)
+  | where toint(HttpStatusCode) >= 400 and toint(HttpStatusCode) <= 499

```

Comment:
same on toint()

Reply:
Query works in Sentinel w/o casting toint().
Casting toint() was added to pass checks. If there is no casting toint() in files then checks fail with error